City-of-Helsinki/drupal-hdbt
# UHF-8277: Fix errors with cookie compliance message

# [UHF-8277](https://helsinkisolutionoffice.atlassian.net/browse/UHF-8277)
Fix errors caused by calling a unknown function.

## What was done
Replaced call to deleted function to prevent errors.
Fixed some lacking styles.

## How to install

* Set up [KASKO](https://github.com/City-of-Helsinki/drupal-helfi-kasvatus-koulutus) instance.
* Add `ELASTIC_PROXY_URL=https://kasvatus-elastic-proxy-test.agw.arodevtest.hel.fi` to your `.env` file to (this way you don't have to set up local elastic)
* Run `make up`
* Install this branch: `composer require drupal/hdbt:UHF-8277-cookie-compliance-errors`
* Run `drush cr`

## How to test
* Add or edit a landing page to include a `School search` paragraph
* View the page in **incognito** mode. Don't accept cookies!
  * The page should load without errors
  *  When pressing the 'view results as a map' -button, you should see a message telling you to either move to the external site or accept cookies.



[UHF-8277]: https://helsinkisolutionoffice.atlassian.net/browse/UHF-8277?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
